core/blockchain_test.go

```diff
@@ -355,11 +355,12 @@ func TestBlockChainOfflinePruningUngracefulShutdown(t *testing.T) {
 			return blockchain, nil
 		}
 
-		tempDir := t.TempDir()
 		if err := blockchain.CleanBlockRootsAboveLastAccepted(); err != nil {
 			return nil, err
 		}
+		blockchain.Stop()
```

Comment:
is this the right place to call `Stop()` if we are going to use `blockchain.LastAcceptedBlock().Root()` below?

Seems like this works because it returns an in-memory value